docs/devsite-help/troubleshooting.md

```diff
@@ -260,3 +260,56 @@ catch (RpcException ex)
     }
 }
 ```
+
+## How can I diagnose proxy issues?
+
+Proxies can be configured in multiple places in .NET:
+
+- In code (e.g. with `HttpClientHandler.Proxy` or `WebRequest.DefaultWebProxy`)
+- In environment variables
+- In system settings
+
+Additionally, the client libraries make multiple requests:
+
+- Auth can sometimes require requests (e.g. to obtain OAuth access tokens, or
+  to access the Compute metadata server); these always use `HttpClient`
```

Comment:
Mentioning Compute metadata server here might be confusing. Not sure that could ever be behind a proxy?

Reply:
It might not actually *be* behind a proxy, but if a proxy has been configured globally then we'd potentially attempt to contact it *through* the proxy - which would presumably fail.